tools/ci_build/github/linux/docker/scripts/install_onnx.sh

```diff
@@ -27,7 +21,6 @@ for onnx_version in ${!version2tag[@]}; do
   cd /tmp/src/onnx-$onnx_version
   git clone https://github.com/pybind/pybind11.git third_party/pybind11
   /usr/bin/python${PYTHON_VER} -m pip install .
-  mkdir -p /data/onnx/${version2tag[$onnx_version]}
-  backend-test-tools generate-data -o /data/onnx/${version2tag[$onnx_version]}
-  echo $onnx_version":"${version2tag[$onnx_version]} >> /data/onnx/version2tag
+  mkdir -p /data/onnx/${onnx_tag}
```

Comment:
>${onnx_tag} [](start = 22, length = 11)

This was a commit previously, is it correct? Is there any other scripts that may depend on the behavior? Is this necessary to change it from commit to a tag?

Reply:
Yes, it was commit id previously.
Only install_deps.sh depends on it.
And yes, we discussed this - tag each commit with a name and use name as filter in main.cc.